documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    !cd {DOWNLOAD_LOCATION}; unzip -qq \*.zip; rm -f *.zip
</pre><p>Running this cell twice causes <code>unzip</code> to prompt through stdin, which  we can't do in colab.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>